api-service/crowdsource-webapp/src/app/base-job/base-job.component.ts

```diff
@@ -173,7 +173,7 @@ export class BaseJobComponent<T> implements OnInit {
       return;
     }
     console.log(data);
-    const randomItemIndex = getRandomInt(0, data.length - 1);
+    const randomItemIndex = getRandomInt(0, data.length);
```

Comment:
nice fix; can't have been easy to catch. :)

Reply:
I only caught it because I was working with a test set of two examples, and wondering why it wasn't working!